notebooks/MAST/plot_exoplanet_lightcurves/plot_exoplanet_lightcurves.ipynb

```diff
@@ -0,0 +1,1678 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    sci_prod_arr = [x for x in obs_products['data'] if x.get("productType", None) == 'SCIENCE']
</pre><p>Can we filter for science products from the beginning? That would make it easier to look through the results</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Hmm, poking around it seems like I can't incorporate this into the actual request. I'll take a closer look at the documentation, though!</p>